### PR TITLE
Add a demo for the `Trim` on the `BitTextField` demo page (#2044)

### LINKED ISSUE #2044: Add a demo for the `Trim` on the `BitTextField` demo page
Improve the `BitTextField` demo page. Add a demo for the `Trim` parameter and add a description about it on the `BitTextField` demo page.

### PR DESCRIPTION
**Pull request by saeedhalab**
This closes #2044

### CODE REVIEW

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor [hunk 1]
```diff
@@ -212,4 +212,18 @@
...
+                         CSharpSourceCode="@example14CSharpCode"
+                         ExampleId="example14">
+        <ExamplePreview>
+            <div class="example-desc">The trim property removes whitespace from both sides of a string.</div>
+            <BitTextField Trim="true"
+                          Label="With Trim"
+                          @bind-Value="TrimTextvalue"></BitTextField>
+            <BitTextField @bind-Value="TextValue1"></BitTextField>
+            <pre>type with trim :@TrimTextvalue</pre>
+            <pre>type without trim :@TextValue1</pre>
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor:226]: please change the `pre` tag like this to be more clear:

```
<pre>type with trim: [@TrimTextvalue]</pre>
<pre>type without trim: [@TextValue1]</pre>
```

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor.cs [hunk 2]
```diff
@@ -9,6 +9,8 @@ public partial class BitTextFieldDemo
     {
         private BitTextFieldType InputType = BitTextFieldType.Password;
         private string TextValue;
+        private string TextValue1;
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor.cs:12]: please rename this field to `NonTrimmedTextValue`

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor.cs [hunk 3]
```diff
@@ -9,6 +9,8 @@ public partial class BitTextFieldDemo
...
+        private string TrimTextvalue;
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor.cs:13]: please rename this field to `TrimmedTextValue`

**Review by msynk — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
@maryamhdr & @Cyrus-Sushiant plz review

**Review by msynk — APPROVED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor.cs [hunk 4]
```diff
@@ -482,6 +491,17 @@ public class ValidationTextFieldModel
     public string RangeText { get; set; }
 }";
 
+        private readonly string example14HTMLCode = @"<div class=""example-desc"">The trim property removes whitespace from both sides of a string.</div>
```
Comment (maryamhdr) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor.cs:494]: Plz modify the desc:
`The trim property removes whitespaces from both sides of a string.`

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor [hunk 5]
```diff
@@ -212,4 +212,18 @@
             }
         </ExamplePreview>
     </ComponentExampleBox>
+    <ComponentExampleBox Title="Trim TextField"
+                         HTMLSourceCode="@example14HTMLCode"
+                         CSharpSourceCode="@example14CSharpCode"
+                         ExampleId="example14">
+        <ExamplePreview>
+            <div class="example-desc">The trim property removes whitespace from both sides of a string.</div>
```
Comment (maryamhdr) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor:220]: Plz modify the desc:
`The trim property removes whitespaces from both sides of a string.`

**Review by maryamhdr — CHANGES_REQUESTED** [review]

**Review by msynk — APPROVED** [review]

**Review by maryamhdr — APPROVED** [review]

**Pull request merged**